src/main/java/com/timelogr/enterprise/Employee.java

```diff
@@ -0,0 +1,25 @@
+package com.timelogr.enterprise;
+
+import lombok.Data;
+
+public @Data
+class Employee {
+
+    private long employeeID;
+    private String employeeName;
+    private String employeeEmail;
+    private String employeeNumber;
```

Comment:
Ditto above.  Good looking class, but remove employee from the variable names.  That's a given, from the class name.